source/win-main/MainSidebar.vue

```diff
@@ -26,7 +26,12 @@
         <div class="toc-level">
           {{ entry.renderedLevel }}
         </div>
-        <div class="toc-entry" v-bind:data-line="entry.line" v-html="entry.text"></div>
+        <div
+          class="toc-entry"
```

Comment:
You can add that to the v-bind directive by using `toc-entry: true` -- this way there is just one class property!

Reply:
Done!